### DISCUSSION TITLE
locally installed Llama2 7b hf

### DISCUSSION THREAD

**Discussion by georgejoy88** [ludwig-ai/ludwig#3702] [General]
Hi - i tried to load a locally installed llama2 7b hf rather than from the huggingface website but the gpu killed the operation.  Please could I check if this is even possible?

**Comment by Infernaught** [comment]
What is the command you are running to load in the local model?

**Reply by georgejoy88** [reply]
i just replaced the huggingface destinationpath for the model with a local destinationpath?  


# zero_shot_config = yaml.safe_load(
#   """
#   model_type: llm
#   base_model: meta-llama/Llama-2-7b-hf

**Reply by Infernaught** [reply]
Ahh, I see. Is this local model a pretrained adapter model or is it just the base llama-2-7b-hf that has been downloaded locally?
